src/components/crag/cragTable.tsx

```diff
@@ -199,51 +188,13 @@ export default function CragTable (props: CragTableProps): JSX.Element {
           </div>
         )}
       </div>
-
-      <DragDropContext
-        onDragEnd={(result) =>
-          setSortedRoutes(
-            reorderFromDrag(
-              sortedRoutes,
-              result.source.index,
-              result.destination?.index ?? 0
-            )
-          )}
-      >
-        <Droppable droppableId='cragTable'>
-          {(provided) => (
-            <div
-              {...provided.droppableProps}
-              ref={provided.innerRef}
-              className={`grid grid-cols-1 lg:gap-3 ${
-                isEditing ? '' : 'xl:grid-cols-4 lg:grid-cols-3 md:grid-cols-2'
-              }  fr-2`}
-            >
-              {sortedRoutes.map((i, idx) => (
-                <Draggable
-                  isDragDisabled={!isEditing}
-                  draggableId={i.id}
-                  index={idx}
-                  key={i.id}
-                >
-                  {(provided, snapshot) => (
-                    <div
-                      className={`m-1 ${
-                        snapshot.isDragging ? 'bg-purple-100' : 'bg-white'
-                      }`}
-                      ref={provided.innerRef}
-                      {...provided.draggableProps}
-                      {...provided.dragHandleProps}
-                    >
-                      <ClimbItem favs={favs} hideSummary={isEditing} key={i.id} climb={i} />
-                    </div>
-                  )}
-                </Draggable>
-              ))}
-            </div>
-          )}
-        </Droppable>
-      </DragDropContext>
+      <div className='grid grid-cols-1 lg:gap-3 xl:grid-cols-4 lg:grid-cols-3 md:grid-cols-2 fr-2'>
```

Comment:
Stripped out the drag and drop functionality, leaving it just as a read-only list.